lib/tools/node-modules/node-modules-builder.ts

```diff
@@ -126,6 +126,14 @@ export class NodeModulesBuilder implements INodeModulesBuilder {
 		let dependenciesBuilder = this.$injector.resolve(NodeModulesDependenciesBuilder, {});
 		let productionDependencies = dependenciesBuilder.getProductionDependencies(this.$projectData.projectDir);
 
+		let prodDependenciesArr = [];
+		for (let i in productionDependencies) {
+			prodDependenciesArr.push(productionDependencies[i].name);
+		}
+
+		let prodDependenciesFilePath = path.join(this.$projectData.projectDir, "platforms", "android", "productionDependencies.json");
+		this.$fs.writeJson(prodDependenciesFilePath, prodDependenciesArr);
```

Comment:
Why gradle needs to know which are the production dependencies? As far as I remember, CLI copies only production dependencies to `platforms/android` dir.
Also... this code is executed for iOS builds as well.